weeklyChallenges01/Program.cs

```diff
@@ -1,8 +1,70 @@
 ﻿using System;
-namespace weeklyChallenges01
+using System.Collections.Generic;
+using System.Text;
+
+namespace ChallengesWithTests01
 {
-    public class Program
+    public class ChallengesSet01
     {
-        static Main 
+        public bool AreTwoNumbersTheSame(int num1, int num2)
+        {
+            return num1 == num2;
+
+
+        }
+
+        public double Subtract(double minuend, double subtrahend)
+        {
+            return minuend - subtrahend;
+
+
+        }
+
+        public int Add(int number1, int number2)
+        {
+            return number1 + number2;
+
+
+        }
+
+        public int GetSmallestNumber(int number1, int number2)
+        {
+            if (number1 < number2)
+            {
+                return number1;
+
+            }
+
+            else return number2;
```

Comment:
watch formatting syntax:
(should be)

`else
{
    return number2
}`

Reply:
not like it put it on that comment but you know, brackets like on the if-statement